### PR TITLE
Database refactor, consistent apollo versions & pubsub await for link subscriptions

### LINKED ISSUE #323: AD4M websocket connection does not crash when fetching new changes from the DHT

### LINKED ISSUE #324: We have a new local database in ad4m-executor using a more stable library and pattern

### PR DESCRIPTION
**Pull request by jdeepee**
- Closes #323 
- Closes #324

### CODE REVIEW

File: executor/src/tests/bootstrapSeed.json
Comment (lucksus) [inline executor/src/tests/bootstrapSeed.json:1]: This file shouldn't be committed, right?

File: executor/src/tests/publishBootstrapSeed.json
Comment (lucksus) [inline executor/src/tests/publishBootstrapSeed.json:1]: Same here.

**Review by lucksus — CHANGES_REQUESTED** [review]
Awesome! Just the two bootstrap seed files that were committed with the language code in it should be reverted.

**Review by lucksus — APPROVED** [review]
Ok, I've reverted the bootstrap seed files. @jdeepee please confirm that those changes were not intended and go ahead and merge this.

**Comment by jdeepee** [comment]
> Ok, I've reverted the bootstrap seed files. @jdeepee please confirm that those changes were not intended and go ahead and merge this.

Ooo yep that was a mistake, will wait to confirm that CI passes since there might still be issue with windows test

**Pull request merged**